fvm/Cargo.toml

```diff
@@ -47,7 +47,7 @@ pretty_assertions = "1.2.1"
 fvm = { path = ".", features = ["testing"], default-features = false }
 
 [dependencies.wasmtime]
-version = "2.0.0"
+version = "0.40.0"
```

Comment:
Nope, doesn't work. I think because the builtin actor bundle v7 is made against ref-fvm 0.6 that still references the 0.40.0, cargo won't allow upgrading to anything with a higher major version.

Reply:
Hm. That's strange. The builtin actors shouldn't care.

But this is just for testing anyways so it doesn't really matter.

Reply:
I'm only guessing, I see this when I try to run the conformance tests:
```
error[E0053]: method `limiter_mut` has an incompatible type for trait
   --> testing/conformance/src/vm.rs:292:5
    |
292 |     fn limiter_mut(&mut self) -> &mut dyn wasmtime::ResourceLimiter {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected trait `wasmtime::limits::ResourceLimiter`, found trait `ResourceLimiter`
    |
    = note: expected fn pointer `fn(&mut TestCallManager<C>) -> &mut dyn wasmtime::limits::ResourceLimiter`
               found fn pointer `fn(&mut TestCallManager<C>) -> &mut dyn ResourceLimiter`
    = note: perhaps two different versions of crate `wasmtime` are being used?
```

I'm not sure if it has anything to do with the `&mut dyn` stuff. I'll try if using associated type makes any difference.

Reply:
Indeed it works, upgraded again https://github.com/filecoin-project/ref-fvm/pull/1036 :tada:

Reply:
Okay, found it. I didn't notice that the integration and the conformance tests both declare `wasmtime` as their own dependencies, that's what caused the duplicate. Should have checked the dependency tree. 

Maybe we could make use of [workspace inheritance](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table) to keep versions in a single place.